core-bundle/src/Routing/Content/ContentUrlResult.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Content;
+
+use Contao\CoreBundle\Exception\ForwardPageNotFoundException;
+use Contao\PageModel;
+use Nyholm\Psr7\Uri;
+
+final class ContentUrlResult
+{
+    private bool $redirect = false;
+
+    public function __construct(public readonly object|string|null $content)
```

Comment:
```suggestion
    public function __construct(public readonly object|string$content)
```
This would be what we agreed on, right?

Reply:
removed in f5fe0239ac967213a8a33fbb5263b364b942086c

Reply:
No, that would not work. Both the `redirect()` and the `resolve()` factory methods allow `null`. So the type hint has to remain.